src/shell/lisa_shell

```diff
@@ -308,19 +319,35 @@ nosetests -v --nocapture --nologcapture \
 }
 
 function lisa-test {
-CMD=${1:-help}
-echo
-case "x${CMD^^}" in
-"xHELP")
-	_lisa-test-usage
-	;;
-*)
-	_lisa-test $*
-        local retcode=$?
-esac
-echo
+local iterations=0 # Means use default - see libs/utils/test.py
+local have_more_args=no
+
 echo
-return $retcode
+while [[ $# -gt 0 ]]; do
+        case "$1" in
+                help|-h|--help)
+                        _lisa-test-usage
+                        return 0
+                        ;;
+                -n|--iterations)
+                        iterations="$2"
+                        shift
+                        ;;
+                *)
```

Comment:
Hmm yeah that's true this doesn't work if you want to pass extra args to nosetests - that's probably what my rationale for the previous implementation was. Will see what I can come up with...